packages/components/src/Dropzone/sub-components/DefaultMessage/DefaultMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Paragraph } from "@kaizen/typography"
+import { Hyperlink } from "../Hyperlink/Hyperlink"
+
+interface DefaultMessageProps {
+  handleOpenDialog: () => void
+}
+
+const DefaultMessage = ({ handleOpenDialog }: DefaultMessageProps): JSX.Element => (
+  <div className="flex flex-col items-center justify-center">
+    <span className="inline-block mb-8">
+      <Paragraph variant="body">
+        <span className="text-purple-800">Drag and drop file or{" "}</span>
+        <Hyperlink text="Select a file" handleOnClick={handleOpenDialog} />
+      </Paragraph>
+    </span>
+    <Paragraph variant="extra-small"><span className="text-purple-800/70">.xlsx format only</span></Paragraph>
```

Comment:
We recommend that this should be plain text from the consumer as it would be difficult to generalise this statement when designers may want to abbreviate this message to say "All images" or not say anything at all when it comes to allowing all file types.

Reply:
As a side note, you can use the `color` and `tag` props on `Paragraph` to achieve these styles.
```suggestion
    <Paragraph variant="extra-small" tag="span" color="dark-reduced-opacity">.xlsx format only</Paragraph>
```